lib/cisco_node_utils/evpn_multisite.rb

```diff
@@ -34,17 +34,31 @@ def initialize(multisiteid, instantiate=true)
     end
 
     def create
-      Feature.nv_overlay_enable
+      # There is an issue with EX platform using
+      # Feature.nv_overlay_enable generating error code '400'
+      # Using config_set for enabling nv overlay
+      config_set('feature', 'nv_overlay', state: '')
+      sleep 1
       @set_args[:state] = ''
       config_set('evpn_multisite', 'multisite', @set_args)
     end
 
     def destroy
       @set_args[:state] = 'no'
+      # HACK: set time to a dummy value
```

Comment:
What are we working around in this case?

Reply:
for configuring evpn_multisite, the command is as follows:

evpn multisite border-gateway 50.

To unconfigure, the command 'no evpn multisite border-gateway' doesn't work. It needs a multisite id which can be any integer. Ideally, the user while writing the maniest needn't have a multisite id while uncofiguring (ensure => absent). Hence, the hack.